source/customizations.rst

```diff
@@ -781,12 +781,8 @@ that lists all user quotas. The JSON schema for version `1` is given as:
      "version": 1,
      "timestamp": 1525361263,
      "quotas": [
-       {
-         ...
-       },
-       {
-         ...
-       }
```

Comment:
Since this isn't valid json, it's incompatible with sphinx 2.4 (we're currently using sphinx 1.8).